logging/src/main/java/org/jboss/as/logging/logging/LoggingLogger.java

```diff
@@ -840,7 +840,7 @@
      * @return an {@link OperationFailedException} for the error
      */
     @Message(id = 79, value = "Failed to read the log file '%s'")
-    RuntimeException failedToReadLogFile(@Cause Throwable cause, String name);
+    OperationFailedException failedToReadLogFile(@Cause Throwable cause, String name);
```

Comment:
Why the change? If the failure is due to user error (bad input) the OFE is fine, but if it's a server fault, throw a RuntimeException.